services/workflows-service/scripts/workflows/kyc-email-process-example.ts

```diff
@@ -63,7 +63,7 @@ export const kycEmailSessionDefinition = {
       {
         name: 'kyc_session',
         pluginKind: 'kyc-session',
-        url: `{secret.KYC_API_URL}/individual-verification-sessions`,
+        url: `${env.UNIFIED_API_URL}/individual-verification-sessions`,
```

Comment:
please validate that it's set in hashicorp and delete all KYC API URL and KYB API URL from all vaults please

Reply:
I will delete the other vars only when it will be merged to prod